src/wp-admin/includes/class-wp-list-table.php

```diff
@@ -1503,6 +1503,78 @@ public function ajax_response() {
 		die( wp_json_encode( $response ) );
 	}
 
+	/**
+	 * Generates views links.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param array $link_data {
+	 *     An array of link data.
+	 *
+	 *     @type string $url     The link URL.
+	 *     @type string $label   The link label.
+	 *     @type bool   $current Optional. Whether this is the currently selected view.
+	 * }
+	 * @return array An array of link markup. Keys match the $link_data input array.
+	 */
+	protected function get_views_links( $link_data = array() ) {
```

Comment:
The list of what is `public` and `protected` in this class is a little random. Was there a reason you went with `protected` here?

Actual question, not a passive change request ;)

Reply:
Yep - the `get_views()` methods in each of the List Tables are set to `protected`, so I just used the same for `get_views_links()`.

Reply:
Thanks, that makes sense.